### PR TITLE
global: move to main as the default branch

### PR DESCRIPTION
**Pull request by david-caro**
Bug: T308968

### CODE REVIEW

**Comment by david-caro** [comment]
Had to recreate this after the rename xd

**Comment by david-caro** [comment]
Merging as I had already a review, and this will block any other merges (due to the rename having been done)

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
   flake8_py3:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@main
```
Comment (supertassu) [inline .github/workflows/lint.yaml:10]: I think this was actually referring to the branch name of https://github.com/actions/checkout? Anyhow doesn't seem to have broken anything

Comment (david-caro) [reply .github/workflows/lint.yaml:10]: Yes, that was intended actually, mentioned in the original PR that got automatically closed with the rename :/

(#157)

Thanks for pointing it out though :+1:

**Pull request merged**